client/src/state-management/Reducer.jsx

```diff
@@ -17,7 +17,7 @@ const Reducer = (state, action) => {
 			};
 		default:
```

Comment:
@PWoram It looks like your case is after the default case which should come last in the order.

Reply:
Thanks!